draft-ietf-quic-http.md

```diff
@@ -723,7 +723,7 @@ with its 0-RTT data.
 ### PUSH_PROMISE {#frame-push-promise}
 
 The PUSH_PROMISE frame (type=0x05) is used to carry a promised request header
-set from server to client, as in HTTP/2.
+set from server to client on a request stream, as in HTTP/2.
```

Comment:
Isn't the use of a PUSH_PROMISE frame also possible on a Push Stream?

Reply:
I don't believe so.  I guess it could, but then you would have to propagate the push to all the streams that referenced that push with PUSH_PROMISE/DUPLICATE_PUSH.

Reply:
Thank you for the clarification. I think having the restriction is fine, because now I realize that RFC 7540 has the same restriction.

Reply:
Right -- same as 7540.  The reason for this is that once a request stream is closed, you know the full set of pushes that are expected in relation to that request.  However, if you allow new promises on push streams, you have to wait until all pushes are satisfied to get a complete set.

I'd be open to a discussion on loosening that requirement, but it's unrelated to QUIC -- write an HTTPbis draft advocating the change if you want it.  :-)